osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -213,6 +212,19 @@ private void updateStatistics()
             displayMetrics(requestedBeatmap.Metrics, false);
         }
 
+        private void loadDetailsAsync(BeatmapInfo beatmap)
+        {
+            if (description == null || source == null || tags == null)
+                throw new InvalidOperationException($@"Requires all {nameof(MetadataSection)} elements to be non-null.");
+
+            Schedule(() =>
+            {
+                description.Text = Beatmap?.Version;

```

Comment:
i'm pretty sure beatmap can never be null

Reply:
why is this accessing `Beatmap` when we have a parameter `beatmap`?

Reply:
The `Beatmap` / `beatmap` thing was just a typo. 
I never planned on actually using this private method after failing with an earlier commit. Will revert to the previous version of doing things, which is just assigning `null` to all text elements when `clearStats` is called and assigning the right text when `updateStatistics` is called